__tests__/components/stake/modals/StakingUnstakeModal.test.tsx

```diff
@@ -99,10 +99,8 @@ describe('Staking Unstake Modal', () => {
         const numberInput = getByRole('spinbutton');
 
         fireEvent.focus(numberInput);
-        await userEvent.type(numberInput, '10000');
-        await waitFor(() => expect(numberInput).toHaveValue('10000'), {
-            timeout: 100,
-        });
+        await userEvent.type(numberInput, '10', {});
+        await waitFor(() => expect(numberInput).toHaveValue('10'));
```

Comment:
This test was failing in CI but passing locally. It appears that in CI it takes significant amount of time for the number input to be filled with the expected value and the given timeout wasn't sufficient. Due to this I tweaked the test slightly.

Reply:
Here's one of the failed builds - https://github.com/cartesi/explorer/actions/runs/17826145920/job/50679668130#step:8:2979. It appears that the test required more time for the number input to be filled with the value of `10000`, so I reduced the value to `10`. The code logic of this test remains unchanged.